src/patterns/exit-the-page-quickly/safety-content-example/index.njk

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
Good news: [Example itself](https://deploy-preview-2721--govuk-design-system-preview.netlify.app/patterns/exit-the-page-quickly/safety-content-example/index.html) looks the way it should

Bad news: I can’t get the embed code to ‘call’ the example on the page to work. It’s also failing some other minor checks.